Packages/com.unity.inputsystem/InputSystem/Plugins/XR/TrackedPoseDriver.cs

```diff
@@ -343,7 +398,17 @@ bool HasStereoCamera()
 
         protected virtual void PerformUpdate()
```

Comment:
(I realize this isn't part of this actual PR, but I wanted to ask about it)
Why is this method virtual? Is there case where the user will want to completely ignore this behavior and do something entirely different instead? Additionally, because this method uses private variables, isn't it functionally impossible for a user to implement a replacement correctly?

Reply:
just allows us to call the same code in update/onbefore render. figured it might be nice for people to override this. but yeah. lack of overall protected et al is probably not great.